src/all/pixiv/src/eu/kanade/tachiyomi/extension/all/pixiv/Pixiv.kt

```diff
@@ -0,0 +1,212 @@
+package eu.kanade.tachiyomi.extension.all.pixiv
+
+import android.util.LruCache
+import eu.kanade.tachiyomi.network.asObservable
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.model.UpdateStrategy
+import eu.kanade.tachiyomi.source.online.HttpSource
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.Jsoup
+import rx.Observable
+import uy.kohesive.injekt.injectLazy
+import java.net.URLEncoder
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+private val json: Json by injectLazy()
+private val dateFormat by lazy { SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.JAPAN) }
```

Comment:
There's nothing in Japanese in the pattern, so better to change it to `Locale.ENGLISH` or `Locale.US`.

```suggestion
private val dateFormat by lazy { SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.ENGLISH) }
```